db/patterns/twitter_button.eno

```diff
@@ -1,8 +1,8 @@
-name: Twitter Button
+name: X Button
 category: social_media
 website_url: http://twitter.com
-organization: twitter
-alias: twitter
+organization: X Corp
+alias: X
```

Comment:
I think, `alias` is also an internal ID and has to stay. (Even though I cannot tell why we have this alias here. Looks redundant to me to define a alias with the same name.)